lte/cloud/go/services/subscriberdb/load_test.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package subscriberdb_test
+
+import (
+	"testing"
+
+	"magma/lte/cloud/go/lte"
+	lte_protos "magma/lte/cloud/go/protos"
+	lte_models "magma/lte/cloud/go/services/lte/obsidian/models"
+	"magma/lte/cloud/go/services/subscriberdb"
+	"magma/lte/cloud/go/services/subscriberdb/obsidian/models"
+	"magma/orc8r/cloud/go/services/configurator"
+	storage2 "magma/orc8r/cloud/go/storage"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestConvertSubEntsToProtos is a regression test to check if ConvertSubEntsToProtos
+// properly filters for the apn associations of a subscriber.
+func TestConvertSubEntsToProtos(t *testing.T) {
```

Comment:
Did this fail with the original implementation?

Reply:
yep! @hcgatewood